src/clarity-angular/wizard/wizard.html

```diff
@@ -12,7 +12,7 @@
     [clrModalSkipAnimation]="stopModalAnimations"
     [clrModalGhostPageState]="ghostPageState"
     [clrModalOverrideScrollService]="isStatic"
-    clrModalPreventClose="true"
+    [clrModalPreventClose]="true"
```

Comment:
Bug? What bug?

I never write bugs.

This is obviously some kind of... thing... that... needed to be changed to a boolean?